OCR/ocr/api.py

```diff
@@ -0,0 +1,34 @@
+import json
+import cv2 as cv
+import numpy as np
+
+from fastapi import FastAPI, UploadFile, Form
+
+from ocr.services.image_ocr import ImageOCR
+from ocr.services.image_segmenter import ImageSegmenter, segment_by_color_bounding_box
+
+app = FastAPI()
+segmenter = ImageSegmenter(
+    segmentation_function=segment_by_color_bounding_box,
+)
+ocr = ImageOCR()
+
+
+@app.get("/")
+async def hello_world():
```

Comment:
you can probably remove this.

Reply:
I actually added it for health checks

Reply:
I am happy to rename it

Reply:
I can rename it to make its purpose more obvious

Reply:
Done, thanks!